components/Header/index.vue

```diff
@@ -1,8 +1,9 @@
 <template>
   <div class="header">
     <div class="header__container wrapper">
-      <img v-if="$breakpoints.xs" class="header__icon" src="~/assets/logo/logo-dedi.svg" alt="Logo Desa Digital" @click="$router.push('/')">
-      <img v-else class="header__icon-typo" src="~/assets/logo/logo-dedi-typo.svg" alt="Logo Desa Digital Typography" @click="$router.push('/')">
+      <img v-if="isXS" class="header__icon" src="~/assets/logo/logo-dedi.svg" alt="Logo Desa Digital" @click="$router.push('/')">
+      <img v-if="isXS === false" class="header__icon-typo" src="~/assets/logo/logo-dedi-typo.svg" alt="Logo Desa Digital Typography" @click="$router.push('/')">
+      <div v-if="isXS === null" class="header__icon-skeleton" />
```

Comment:
imho I think it's a bit over-engineered but we can move this functionality to a _mixin_ and use it in another component if we need it

Reply:
yes it's a bit complicated and over-engineered , of course many ways to handle that, and thanks for you @bangunbagustapa and @Ibwedagama, i will look for another workaround to make that simple

Reply:
@Ibwedagama I agree, we can abstract it and make it reusable if it is likely to be used.

It's okay if it's not implemented now because many approaches can be used, right now is all good @adzharamrullah ❤️

Reply:
@Ibwedagama @bangunbagustapa thanks for you suggentions. here https://github.com/jabardigitalservice/dedi-landing-page/pull/7/commits/88ac7c2d2c8bcfcb431917ba88e7a5cdc5f3eb3c more it simple